### PR TITLE
Update CHARTER.md

### PR DESCRIPTION
**Pull request by kaywilliams**
Remove duplicate wording.

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Effective 4 February 2019
 
 #### 1) Mission and Scope of the Continuous Delivery Foundation (the "Foundation," or, alternatively, the “Directed Fund”).
 
-- a)  The purpose of the Foundation is to raise, budget and spend funds in support of various open source and/or open standards projects relating to raise, budget and spend funds in support of various continuous delivery open source and standards projects, including infrastructure and support related thereto, including infrastructure and support initiatives related thereto (each such project, a "Technical Project"). The governance of each Technical Project is as set forth in the applicable charter for each Technical Project.
+- a)  The purpose of the Foundation is to raise, budget and spend funds in support of various open source and/or open standards projects  in support of various continuous delivery open source and standards projects, including infrastructure and support related thereto, including infrastructure and support initiatives related thereto (each such project, a "Technical Project"). The governance of each Technical Project is as set forth in the applicable charter for each Technical Project.
```
Comment (stmcginnis) [inline CHARTER.md:9]: Extremely trivial nit:

s/projects  in/projects in/

Comment (stmcginnis) [reply CHARTER.md:9]: I think the two "in support of" may be a little redundant too. Maybe something like "funds supporting various open source"?

**Review by stmcginnis — COMMENTED** [review]
According to [1], this will need to have a formal vote by the governing board.

[1] https://lists.cd.foundation/g/cdf-toc/message/20

**Comment by tracymiranda** [comment]
These fixes were incorporated in https://github.com/cdfoundation/charter/pull/8

**Pull request closed without merging**